gtsfm/scene_optimizer.py

```diff
@@ -394,21 +405,19 @@ def aggregate_frontend_metrics(
     )
 
     logger.debug(
-        "[Two view optimizer] [Summary] Pose success: %d/%d/%d", success_count_pose, num_valid_entries, num_entries,
+        "[Two view optimizer] [Summary] Pose success: %d/%d/%d", success_count_pose, num_valid_entries, num_entries
     )
 
-    logger.debug(
-        "[Two view optimizer] [Summary] Image pairs with 100%% inlier ratio:: %d/%d", all_correct, num_entries,
-    )
+    logger.debug("[Two view optimizer] [Summary] Image pairs with 100%% inlier ratio:: %d/%d", all_correct, num_entries)
 
     front_end_result_info = {
         "angular_err_threshold_deg": angular_err_threshold_deg,
         "num_valid_entries": int(num_valid_entries),
         "num_total_entries": int(num_entries),
-        "rotation": {"success_count": int(success_count_rot3),},
-        "translation": {"success_count": int(success_count_unit3),},
-        "pose": {"success_count": int(success_count_pose),},
-        "correspondences": {"all_inliers": int(all_correct),},
+        "rotation": {"success_count": int(success_count_rot3)},
+        "translation": {"success_count": int(success_count_unit3)},
+        "pose": {"success_count": int(success_count_pose)},
+        "correspondences": {"all_inliers": int(all_correct)},
     }
 
     io_utils.save_json_file(os.path.join(METRICS_PATH, "frontend_summary.json"), front_end_result_info)
```

Comment:
I noticed you were removing a lot of trailing commas that weren't needed. On lines 89 and 256, I think you have trailing commas which could be removed.

Reply:
Thanks for taking a look Adi. Python black adds those sometimes -- the ones I removed earlier I believe were due to us formatting and then re-formatting with different options, that led to the artifact being added.

I think the ones in lines 89 and 256, black will force us to keep for now